CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ Pipeline has been re-implemented in [Nextflow DSL2](https://www.nextflow.io/docs
 - [#29](https://github.com/nf-core/metapep/pull/29) - Added data model figure to `output.md`
 - [#45](https://github.com/nf-core/metapep/pull/45) - Added support for multiple weights tables for one bin (i.e. co-assembly input)
 - [#56](https://github.com/nf-core/metapep/pull/56),[#61](https://github.com/nf-core/metapep/pull/61) - Updated documentation
-- [#67](https://github.com/nf-core/metapep/pull/67) - Added parameters to adjust binder/non-binder calling. Additional documentation on scoring by `SYPEITHI`, `MHCflurry`and `MHCnuggets`
+- [#67](https://github.com/nf-core/metapep/pull/67), [#78](https://github.com/nf-core/metapep/pull/78) - Added parameters to adjust binder/non-binder calling. Additional documentation on scoring by `SYPEITHI`, `MHCflurry`and `MHCnuggets`. Additional Parameters for pandas memory usage logging and a global chunk size parameter.
```

Comment:
```suggestion
- [#67](https://github.com/nf-core/metapep/pull/67) - Added parameters to adjust binder/non-binder calling. Additional documentation on scoring by `SYPEITHI`, `MHCflurry`and `MHCnuggets`.
[#78](https://github.com/nf-core/metapep/pull/78) - Added parameter for pandas memory usage logging.
[#78](https://github.com/nf-core/metapep/pull/78) - Added a global chunk size parameter.
```

Reply:
Okay, I think in this case I had thematic changeling wrong in my head! I was on parameter changes as open topic, also explains the larger PR with multiple real topics! Sorry about that!